app/views/reviews/new.html.erb

```diff
@@ -1,3 +1,9 @@
+<% flash.each do |type, msg| %>
+  <div>
+    <font size="5" color="red"><%= msg %></font>
+  </div>
+<% end %>
+
```

Comment:
I mentioned that we may want to house the flash messages in the application folder, but I am okay with housing them each individually in their respective views. I like your styling better anyways, so I am following your convention here.

Reply:
Sounds good, we can change it as we go.